### ISSUE TITLE
Release notes

### ISSUE THREAD

**Issue by JulioCM** [davidcarslaw/openair#99]
I wonder if there is a way to check the updates and fixes of each new release?

**Comment by davidcarslaw** [comment]
Yes, the best place to bookmark openair is http://davidcarslaw.github.io/openair/. You will see a NEWS section at the top, which contains information on (nearly) all changes that have been made by version. When I have enough time I plan to move the openair manual to the website too and having an online 'book' using bookdown (https://bookdown.org/). Hopefully that is useful to you...

**Comment by jack-davison** [comment]
> When I have enough time I plan to move the openair manual to the website too and having an online 'book' using bookdown (https://bookdown.org/). Hopefully that is useful to you...

Closing, and mentioning that this book now does exist at: https://bookdown.org/david_carslaw/openair/
